base/dates/periods.jl

```diff
@@ -65,27 +65,34 @@ Base.isless{P<:Period}(x::P,y::P) = isless(value(x),value(y))
 =={P<:Period}(x::P,y::P) = value(x) == value(y)
 
 # Period Arithmetic, grouped by dimensionality:
-import Base: div, mod, rem, gcd, lcm, +, -, *, /, %, .+, .-, .*, .%
+import Base: div, fld, mod, rem, gcd, lcm, +, -, *, /, %, .+, .-, .*, .%
 for op in (:+,:-,:lcm,:gcd)
     @eval ($op){P<:Period}(x::P,y::P) = P(($op)(value(x),value(y)))
 end
 
-for op in (:/,:%,:div,:mod)
+for op in (:/,:div,:fld)
     @eval begin
         ($op){P<:Period}(x::P,y::P) = ($op)(value(x),value(y))
         ($op){P<:Period}(x::P,y::Real) = P(($op)(value(x),Int64(y)))
     end
 end
+
+for op in (:rem,:mod)
+    @eval begin
+        ($op){P<:Period}(x::P,y::P) = P(($op)(value(x),value(y)))
```

Comment:
I intentionally left out that method definition as I was unsure whether that was actually a real use case for `rem(Second(5), 2)`. I think I'd prefer to be cautious and leave it out for now.